docs/source/1.0/spec/core/http-traits.rst

```diff
@@ -673,7 +673,7 @@ structure with the ``httpLabel`` trait MUST have a corresponding
   ``1985-04-12T23%3A20%3A50.52Z``). The :ref:`timestampFormat-trait`
   MAY be used to use a custom serialization format.
 - Reserved characters defined in :rfc:`section 2.2 of RFC3986 <3986#section-2.2>`
-  and the `%` itself MUST be percent-encoded_ (that is, ``:/?#[]@!$&'()*+,;=%``).
+  and the `%` itself MUST be percent-encoded_ (that is, ``:/?#[]@!$&'()*+,;=%><"`? ``). Note the inclusion of a literal space, ` `.
```

Comment:
At this point it'd probably be shorter to list the things that aren't percent encoded according to RFC-3986. That'd be `ALPHA / DIGIT / "-" / "." / "_" / "~" / ":" / "@" / "!" / "$" / "&" / "'" / "(" / ")" / "*" / "+" / "," / ";" / "="` where ALPHA is valid alphabetic characters in US-ASCII.

Interesting that we've called out `@` and `:` where it's generally allowed in a path segment as-is.